service-loadbalancer/template.cfg

```diff
@@ -74,6 +74,7 @@ frontend httpfrontend
     acl url_{{$svc.Name}} path_beg /{{$svc.Name}}
     use_backend {{$svc.Name}} if url_{{$svc.Name}}
 {{end}}
+    default_backend default-backend
```

Comment:
can we make this a global default_backend? That way we can also have a frontend specific default_backend that overrides the global. If we allow people to request different ports, they can also in theory give us their own 404 page. If they don't and nothing matches, we server the lb 404 page.